dist2src.py

```diff
@@ -144,18 +158,37 @@ def copy_spec(ctx, origin, dest):
 @log_call
 @click.pass_context
 def copy_patches(ctx, origin, dest):
-    """Find and copy patches from a dist-git repo to a source-git repo.
+    """Copy 'SOURCES/*.patch' from a dist-git repo to a source-git repo."""
+    _copy_files(origin=origin, dest=dest, dir="SOURCES", glob="*.patch")
 
-    This looks for 'SOURCES/*.patch' in ORIGIN and copy everything found to
-    'centos-packaging/SOURCES/' in DEST.
-    """
-    orig_sources = Path(origin, "SOURCES")
-    dest_sources = Path(dest, "centos-packaging", "SOURCES")
 
-    dest_sources.mkdir(parents=True, exist_ok=True)
+@cli.command()
+@click.argument("dest", type=click.Path(exists=True, file_okay=False))
+@log_call
+@click.pass_context
+def add_packit_config(ctx, dest):
+    dest_dir = Path(dest)
+    config = {
+        "specfile_path": f"{DOWNSTREAM_FILES_DIR}/SPECS/{dest_dir.name}.spec",
+        "upstream_ref": SG_START_TAG,
+        "jobs": [
```

Comment:
at some point, we should get this behaviour to packit-service so that we don't have any packit.yaml in these source-git repos